forge-gui/res/adventure/Shandalar/custom_cards/giant_scythe.txt

```diff
@@ -0,0 +1,7 @@
+Name:Giant Scythe
+ManaCost:no cost
+Types:Artifact Equipment
+S:Mode$ Continuous | Affected$ Creature.Artifact+YouCtrl | EffectZone$ Command |  AddToughness$ 2 | AddType$ Scarecrow | Description$ Artifact creatures you control get +0/+2 and are Scarecrow in addition to their other types. 
+A:AB$ ChangeZone | Cost$ 3 M M M M Sac<3/Scarecrow/Scarecrows> | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature.YouOwn+Artifact | TgtPrompt$ Select target artifact creature cards from your graveyard | ChangeNum$ 1 | ActivationZone$ Command | SorcerySpeed$ True | SubAbility$ ExileSelf | SpellDescription$ Return target artifact creature from your graveyard to the battlefield tapped. Activate only as a sorcery. Exile Giant Scythe.
```

Comment:
I thought paying shards is like this: PayShards<1>, did you test the card if Cost$ 3 M M M M Sac<3/Scarecrow/Scarecrows> works?